src/experimental/safe_html_transform/tests/test_safe_html.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+import unittest2 as unittest
+from Products.PortalTransforms.data import datastream
```

Comment:
@tisto O.o that is the wrong way I was testing. Sorry for that messup :( . Will go through plone.intelligenttext and will out the correct way to test the safe_html.